sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -496,6 +497,13 @@ private static string GetMarketAsset(LusidInstrument option, MarketQuote.QuoteTy
                 FxOption fxoption = (FxOption) option;
                 marketAsset = $"{fxoption.DomCcy}/{fxoption.FgnCcy}/" + (volType == MarketQuote.QuoteTypeEnum.NormalVol ? "N" : "LN");
             }
+            
+            if (option.InstrumentType == LusidInstrument.InstrumentTypeEnum.InterestRateSwaption)
+            {
+                InterestRateSwaption swaption = (InterestRateSwaption) option;
```

Comment:
VS is complaining that this variable isn't used

Reply:
VS is correct, I added a note below on L504
In the future, it would be good to get the currency from the swaption and generalise the code... just don't know how / can't currently. I want to keep it for when we do the generalisation but happy to remove if you insist otherwise.